docker/airflow/1.10.5/apks/apk-builder/Dockerfile

```
@@ -0,0 +1,22 @@
+# Based off https://github.com/andyshinn/docker-alpine-abuild but for apline 3.5
+FROM alpine:3.10
+RUN apk --no-cache add alpine-sdk coreutils \
+  && adduser -G abuild -g "Alpine Package Builder" -s /bin/ash -D builder \
```

```suggestion
  && adduser -G abuild -g "Alpine Package Builder" -s /bin/sh -D builder \
```
Should this be `/bin/sh`?

---

Could be, they are all symlinks in alpine to /bin/busybox

---

oh, didn't realize alpine had that symlinked.